_includes/nav.html

```diff
@@ -1,11 +1,32 @@
 {% comment %} tag for this page is {% include nav.html %} {% endcomment %}
 
-<div class = 'navbar w3-bar' style='text-align: center'>
-    <a class = 'nav-button w3-bar-block w3-button w3-mobile' href="index.html">Home</a>
-    <a class = 'nav-button w3-bar-block w3-button w3-mobile' href="season-config.html">Season Configuration</a>
-    <a class = 'nav-button w3-bar-block w3-button w3-mobile' href="event-data.html">Event Data</a>
-    <a class = 'nav-button w3-bar-block w3-button w3-mobile' href="team-data.html">Team Data</a>
-    <a class = 'nav-button w3-bar-block w3-button w3-mobile' href="importing-exporting-data.html">Importing and Exporting Data</a>
+{% capture page_list %}
+{% for page in site.data.pages %}
+    <a href= '{{ page.url }}' class="nav-button w3-bar-block w3-mobile w3-button">{{ page.title }}<a>
+{% endfor %}
+{% endcapture %}
+
+<div style='background-color: maroon'>
+    <h1 id = "title">FRC 6321 Scouting</h1>
+    <div class = 'navbar w3-bar' style='text-align: center'>
+        <a class = 'nav-button w3-bar-block w3-button' href="/">Home</a>
+        <span class = 'w3-hide-small'>
+            {{ page_list }}
+        </span>
+        <a href="javascript:void(0)" class="nav-button w3-bar-item w3-button w3-right w3-hide-large w3-hide-medium" onclick="toggleNav()">&#9776;</a>
```

Comment:
To make this work properly, you need to include the following line in the `head` (via head.html)

```html
<meta name="viewport" content="width=device-width, initial-scale=1.0">
```

Docs: https://www.w3schools.com/html/html_responsive.asp

Basically, this tells the browser to adjust the size of the site to fit the screen - this should fix it looking way to small on actual mobile (but working fine when you resize Chrome down - this is caused by difference in resolution sizes).

Reply:
Thanks, this makes the chrome simulator work.